src/init.cpp

```diff
@@ -555,6 +558,7 @@ void SetupServerArgs(ArgsManager& argsman)
     argsman.AddArg("-bytespersigop", strprintf("Equivalent bytes per sigop in transactions for relay and mining (default: %u)", DEFAULT_BYTES_PER_SIGOP), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
     argsman.AddArg("-datacarrier", strprintf("Relay and mine data carrier transactions (default: %u)", DEFAULT_ACCEPT_DATACARRIER), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
     argsman.AddArg("-datacarriersize", strprintf("Maximum size of data in data carrier transactions we relay and mine (default: %u)", MAX_OP_RETURN_RELAY), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
+    argsman.AddArg("-mempoolfullrbf", strprintf("Accept transaction replace-by-fee without replaceability signaling inspection (default: %u)", DEFAULT_MEMPOOL_FULL_RBF), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
```

Comment:
This still needs to be fixed to `-mempoolreplacement=fee,-optin`

Reply:
Personally, I found the string arg confusing over a boolean. If we have more contributors weighting on `-mempoolreplacement`, I'll think about a change.

If you comment is motivated as maintaining config file compatibility across full-node implementations, as I raised on IRC I think the project philosophy isn't really clear on what to do. I invite you to open an issue to document the cross-implementation compatibility config file issues and gather more thoughts on that specific topic.